--- libbind-6.0/include/netdb.h~	2015-06-10 15:00:12.000000000 +0100
+++ libbind-6.0/include/netdb.h	2015-06-10 15:01:03.000000000 +0100
@@ -223,20 +223,16 @@
 #define EAI_MAX		14
 
 /*%
- * Flag values for getaddrinfo()
+ * Flag values for getaddrinfo() and getipnodebyname()
  */
 #define	AI_PASSIVE	0x00000001
 #define	AI_CANONNAME	0x00000002
 #define AI_NUMERICHOST	0x00000004
-#define	AI_MASK		0x00000007
-
-/*%
- * Flag values for getipnodebyname()
- */
 #define AI_V4MAPPED	0x00000008
 #define AI_ALL		0x00000010
 #define AI_ADDRCONFIG	0x00000020
 #define AI_DEFAULT	(AI_V4MAPPED|AI_ADDRCONFIG)
+#define	AI_MASK		0x0000003f
 
 /*%
  * Constants for getnameinfo()
